_posts/troubleshooting/2019-03-29-authn-timeout-401.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2019-03-29
+title: Authentication Timeout Results in 401 Error
+categories:
+   - troubleshooting
+description: SAML authentication timeout results in 401 error
+type: Document
+---
```

Comment:
I think you can set a "forward" here; since you're changing the name of the KB article, it would be nice to be able to use the old URL and still end up "here".